tests/test_diaForcedSource.py

```diff
@@ -151,21 +153,17 @@ def setUp(self):
         self.testDiaObjects = create_test_dia_objects(5, self.wcs)
         # Add additional diaObjects that are outside of the above difference
         # and calexp visit images.
-        # xy outside
-        src = self.testDiaObjects.addNew()
-        src['id'] = 10000000
-        src.setCoord(self.wcs.pixelToSky(-100000,
-                                         -100000))
-        # y outside
-        src = self.testDiaObjects.addNew()
-        src['id'] = 10000001
-        src.setCoord(self.wcs.pixelToSky(100,
-                                         -100000))
-        # x outside
-        src = self.testDiaObjects.addNew()
-        src['id'] = 10000002
-        src.setCoord(self.wcs.pixelToSky(-100000,
-                                         100))
+        points = [
+            self.wcs.pixelToSky(-100000, -100000),  # xy outside
+            self.wcs.pixelToSky(100, -100000),  # y outside
+            self.wcs.pixelToSky(-100000, 100),  # x outside
+        ]
+        extra = pandas.DataFrame({
+            "diaObjectId": np.array([10000000, 10000001, 10000002], dtype=np.int64),
```

Comment:
I'd suggest presenting the ID list in the same form as `points`, so that it's easier to see the parallel.